modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -43,19 +44,19 @@ ContentDashboardAdminConfigurationDisplayContext contentDashboardAdminConfigurat
 		<liferay-frontend:fieldset-group>
 			<liferay-frontend:fieldset>
 				<aui:field-wrapper>
-					<p class="sheet-text">
-						<liferay-ui:message key="select-vocabularies-description" />
-					</p>
+					<span aria-hidden="true" class="loading-animation loading-animation-sm"></span>
```

Comment:
Just out of curiosity: Why was this added here? Is there anyone who manipulates him?

Reply:
The React component takes a little time to render and in the meantime that loading animation shows up. When the component is ready, it will override the content, so the loading animation will be removed from the rendered content (if I'm not wrong).

Reply:
Yeah, it makes sense. Thanks!